arviz/plots/backends/bokeh/bokeh_elpdplot.py

```diff
@@ -173,16 +173,20 @@ def _plot_atomic_elpd(
             )
     if ylabels_shown:
         ax_.yaxis.axis_label = "ELPD difference"
+    else:
+        ax_.yaxis.minor_tick_line_color = None
+        ax_.yaxis.major_label_text_font_size = "0pt"
+
     if xlabels_shown:
         if xlabels:
             ax_.xaxis.ticker = np.arange(0, len(coord_labels))
             ax_.xaxis.major_label_overrides = {
-                key: str(value)
+                str(key): str(value)
```

Comment:
Does this work?

Because ticker value is transformed to string value?

Reply:
I changed it because otherwise it did not work. It seems like it works, but when you try to show the plot you get a typeerror saying dict keys must be strings. Doing this I was able to generate and see the plot with the labels